hack/update-depstat.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+source "${KUBE_ROOT}/hack/lib/init.sh"
+source "${KUBE_ROOT}/hack/lib/util.sh"
+
+kube::golang::verify_go_version
+kube::golang::setup_env
+
+
+echo 'installing depstat'
+pushd "${KUBE_ROOT}/hack/tools" >/dev/null
+  GO111MODULE=on
+  go install github.com/RinkiyaKeDad/dependency-analyzer-poc
```

Comment:
should be one one line!
`GO111MODULE=on go install github.com/RinkiyaKeDad/dependency-analyzer-poc`